common/src/main/java/com/microsoft/identity/common/internal/ui/embeddedwebview/EmbeddedWebViewAuthorizationStrategy.java

```diff
@@ -58,10 +60,12 @@
      * @return generic AuthorizationResult
      */
     public GenericAuthorizationResult requestAuthorization(final GenericAuthorizationRequest request) {
```

Comment:
Consider declaring this method [`@Nullable`](https://developer.android.com/reference/android/support/annotation/Nullable) to convey its potential return value